api/spec/spec_helper.rb

```diff
@@ -36,7 +36,7 @@
 require 'spree/api/testing_support/helpers'
 require 'spree/api/testing_support/setup'
 
-ActiveJob::Base.queue_adapter = :inline
+ActiveJob::Base.queue_adapter = :test
```

Comment:
Have you tried to just remove it?

I think the dummy app generated by Rails will most likely already have set this in the `environments/test.rb`.